Sources/ProjectDescription/TuistConfig.swift

```diff
@@ -1,12 +1,14 @@
 import Foundation
 
 /// This model allows to configure Tuist.
-public class TuistConfig: Codable {
+public class TuistConfig: Encodable, Decodable, Equatable {
     /// Contains options related to the project generation.
     ///
     /// - generateManifestElement: When passed, Tuist generates the projects, targets and schemes to compile the project manifest.
-    public enum GenerationOption: String, Codable {
+    /// - xcodeProjectName(TemplateString): When passed, Tuist generates the project with the specific name on disk instead of using the project name.
+    public enum GenerationOption: Encodable, Decodable, Equatable {
```

Comment:
Renaming `GenerationOption` to `GenerationOptions` is a breaking change but it should be safe because `TuistConfig` hasn't been officially released yet.